### PR TITLE
CHANGELOG for 0.4.3-rc2

### PR DESCRIPTION
**Pull request by Kubuxu**
License: MIT
Signed-off-by: Jakub Sztandera <kubuxu@protonmail.ch>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # go-ipfs changelog
 
+### 0.4.3-rc2 - TODO
+
+This release includes bugfixes and fixes for regressions that were introduced
+between 0.4.2 and 0.4.3-rc1.
+
+- Regressions
+  - Fix daemon panic when there is no multipart input provided with over HTTP API.
```
Comment (RichardLitt) [inline CHANGELOG.md]: "provided over the HTTP API"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+  (@whyrusleeping, [ipfs/go-ipfs#2989](https://github.com/ipfs/go-ipfs/pull/2989))
+  - Fix `ipfs refs` with `edges` option not printing edges.
```
Comment (RichardLitt) [inline CHANGELOG.md]: Fix `ipfs refs --edges` not printing edges.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+  (@Kubuxu, [ipfs/go-ipfs#3007](https://github.com/ipfs/go-ipfs/pull/3007))
+  - Fix progress option for `ipfs add` defaulting to true on http API.
```
Comment (RichardLitt) [inline CHANGELOG.md]: on the HTTP API

**Comment by Kubuxu** [comment]
SGTM

**Pull request merged**